specs/ws-api.yaml

```diff
@@ -143,7 +143,7 @@ components:
             id: 1
```

Comment:
Not related to this file, but `rest-api.yaml` needs to be updated also.

Reply:
done